apiconcepts/integration/overview.md

```diff
@@ -5,15 +5,18 @@ Welcome to the <Var:ProductName> Integration API. This API provides access to <V
 
  <Var:ProductName> Integration API
  -----
-The <Var:ProductName> Integration API enables third-party developers to add or customize new functionalities in Studio applications and more specific to <Var:ProductName> application (for more information see: What you can do with the Integration API).
+The <Var:ProductName> Integration API enables third-party developers to add or customize new functionalities in Studio applications and more specific to <Var:ProductName> application (for more information see: [What you can do with the Integration API](what_you_can_do_with_the_integration_API.md)).
 
 The main topics of the <Var:ProductName> Integration API are:
 
-* User interface integration
-* Studio automation integration
+* [User interface integration](user_interface_integration.md)
+* [Studio automation integration](studio_automation.md)
 
-Before starting to use the <Var:ProductName> Integration API, it is recommended to read the following topics:
+<Var:ProductName> has been developed in a modular fashion, being based on components (called plug-ins) both at the system level (for the built-in features) and as extensions provided by the community and set up by customers. Here is a small diagram illustrating this approach:
+
+![Modularity](images/Modularity.png)
 
-[Setting up a Development Machine](../../articles/gettingstarted/setting_up_a_developer_machine.md) 
+Before starting to use the <Var:ProductName> Integration API, it is recommended to read the following topics:
```

Comment:
Before starting to use the <Var:ProductName> Integration API, it is recommended to read the following topics:
Please change to "We recommend reading the following topics before you start using .... API".